workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/hooks/useTickets.ts

```diff
@@ -16,38 +17,20 @@ const useTickets = ({
 	search,
 }: {
 	filter: {field: string; value: string};
-	page: number;
+	page: string | number;
```

Comment:
allowing both isn't ideal. You should normalize as a single type before passing into the component